udata/tests/test_discussions.py

```diff
@@ -100,6 +100,44 @@ def test_new_discussion_missing_subject(self):
         })
         self.assertStatus(response, 400)
 
+    def test_new_discussion(self):
+        self.app.config['USE_METRICS'] = True
```

Comment:
As you're only testing the extras part, I think you can safely remove everything related to metrics